src/main/java/com/cburch/logisim/std/wiring/PinAttributes.java

```diff
@@ -41,27 +42,36 @@ public boolean isHidden() {
   public static final Attribute<String> ATTR_DUMMY = new DummyAttr("type");
   public static PinAttributes instance = new PinAttributes();
 
-  private static final List<Attribute<?>> ATTRIBUTES =
-      Arrays.asList(
-          StdAttr.FACING,
-          Pin.ATTR_TYPE,
-          StdAttr.WIDTH,
-          Pin.ATTR_TRISTATE,
-          Pin.ATTR_PULL,
-          StdAttr.LABEL,
-          StdAttr.LABEL_FONT,
-          RadixOption.ATTRIBUTE,
-          PROBEAPPEARANCE,
-          ATTR_DUMMY);
+  private List<Attribute<?>> myAttributes;
 
   BitWidth width = BitWidth.ONE;
   boolean threeState = false; // true;
   int type = EndData.INPUT_ONLY;
   Object pull = Pin.PULL_NONE;
+  Long initialValue = 0L;
```

Comment:
Do we need `Long` here and not `long` primitive? If so `var` should be inferred from value so there would not be a need for explicit type declaration.

Reply:
done

Reply:
@BFH-ktt1: At least on GitHub, you don't seem to have taken into account @MarcinOrlowski's suggestion.